scripts/utils/advisory_lock.sh

```diff
@@ -35,4 +35,6 @@ advisory_lock_remove() {
     rm -f "$lock_filepath"
     echo "Removed the lock on the file $filepath."
   fi
-}
\ No newline at end of file
+}
+
+set +o errexit +o errtrace +o pipefail +o history +o allexport
```

Comment:
I'm no shell expert but seems this also can cause issues, if calling script had any of these set it will unset them?

Reply:
I think it is possible. Possibly, We need to judge before setting the flag in the begining and unset it accordingly.

Reply:
If the import on the calling script is after their `set`s, then yes. Should we remove the sets on these utils scripts, so these have no options set, and the calling script controls all options?

Reply:
Removed all set options, this should work best when importing these.